Framework/CurveFitting/inc/MantidCurveFitting/Functions/Bk2BkExpConvPV.h

```diff
@@ -24,7 +24,8 @@ namespace Functions {
 */
 class MANTID_CURVEFITTING_DLL Bk2BkExpConvPV : virtual public API::IPeakFunction, virtual public API::IFunctionMW {
 public:
-  Bk2BkExpConvPV();
+  /// Default constructor.
+  Bk2BkExpConvPV() : API::IPeakFunction() {}
```

Comment:
Just a minor comment, but this isn't necessary as the default constructor for each base class will be implictly called. In this case you can get away without defining a constructor, or explictly defaulting the constructor, e.g

```suggestion
  Bk2BkExpConvPV() = default;
```

Reply:
Thanks for spotting this, I have done as you suggested and explicitly called the default constructor. Also I have added release notes.